public/js/main.js

```diff
@@ -40,6 +41,7 @@ const actions = bindActionCreators(require("./actions"), store.dispatch);
 // global for debugging purposes only!
 window.store = store;
 window.injectDebuggee = require("./test/utils/debuggee");
+window.serializeStore = () => JSON.parse(JSON.stringify(store.getState()));
```

Comment:
I'd rather not keep adding globals here in main.js... would you be ok creating a new module somewhere that does all this injection? Could be something like `require("./utils/debug")(store)` and that module would only inject all this stuff if `process.env.NODE_ENV !== "production"` or something (or whatever debug mode flag).